### DISCUSSION TITLE
seatunnel 对数据血缘的支持

### DISCUSSION THREAD

**Discussion by kissycn** [apache/seatunnel#5353] [Q&A]
Seatunnel 作为大数据平台中非常重要的组件，需要对数据治理生态的支持，例如：数据血缘等。Atlas 作为大数据生态中的数据血缘和数据资产组件已经有非常成熟的血缘采集插件，目前与之对应的数据血缘插件有：Sqoop。
有关 Sqoop 的血缘数据采集是通过[Sqoop Hook](https://github.com/apache/atlas/blob/master/addons/sqoop-bridge/src/main/java/org/apache/atlas/sqoop/hook/SqoopHook.java) 来实现的。
我想咨询下以下几点：
1、Seatunnel 有类似的 [Sqoop Hook](https://github.com/apache/atlas/blob/master/addons/sqoop-bridge/src/main/java/org/apache/atlas/sqoop/hook/SqoopHook.java)  用于采集数据 Source、Sink 的血缘数据吗？
2、社区有计划支持数据血缘采集吗？

**Comment by xyzhh** [comment]
You can build the lineage by parsing SQL.
